python/lsst/obs/sdss/yanny.py

```diff
@@ -38,11 +41,13 @@
 # NOTE: commented out to remove astropy.extern.six dependency
```

Comment:
This commented block can be removed (along with the six.PY3 thing), since we now have py3 support.